crypto/evp/keymgmt_lib.c

```diff
@@ -462,10 +464,8 @@ int evp_keymgmt_util_copy(EVP_PKEY *to, EVP_PKEY *from, int selection)
 
         if (!evp_keymgmt_util_export(from, selection,
                                      &evp_keymgmt_util_try_import,
-                                     &import_data)) {
-            evp_keymgmt_freedata(to_keymgmt, alloc_keydata);
+                                     &import_data))
```

Comment:
Uhm, why?

Reply:
Because the alloc_keydata is always NULL here.